### DISCUSSION TITLE
Is there of a Java version/library that has the same logic as dlt_convert?

### DISCUSSION THREAD

**Discussion by aadi123** [COVESA/dlt-daemon#606] [Q&A] [answered]
For context, I work on an ingestion team at Toyota. I was wondering if there is a java library with the same logic of `dlt_convert.c`?

**Comment by minminlittleshrimp** [comment, marked as answer]
Hello @aadi123 
So sorry that this topic is out of our scope.
You can start a side project for this topic and contribute to DLT if it is possible for you.
Thank you
